scala-projects/game/main/scala/campaign/fight/FightHandler.scala

```diff
@@ -0,0 +1,80 @@
+package main.scala.campaign.fight
+import campaign.characters.Character
+import campaign.enemies.Enemies
+import campaign.spells._
+
+class FightHandler {
+
+  var spellHandling = new SpellHandler
```

Comment:
ez úgy lehetne jobb h ez is appcontextben jön létre, mert így ha máshol kell spellhandler létre kell hoznod még egyet, de abból mivel logika van benne csak egy elég az egész programban